python/hololink/operators/image_processor/image_processor.cpp

```diff
@@ -89,6 +89,8 @@ PYBIND11_MODULE(_image_processor, m)
     py::enum_<ImageProcessorOp::BayerFormat>(op, "BayerFormat")
         .value("RGGB", ImageProcessorOp::BayerFormat::RGGB,
             R"pbdoc(RGGB)pbdoc")
+        .value("BGGR", ImageProcessorOp::BayerFormat::BGGR,
```

Comment:
Since BGGR = 0 in BayerFormat enum, can you add it before RGGB so that the sequence is maintained? Or you can make BGGR = 3in the enum class and add it in the end.

Reply:
Someone mentioned that the BGGR is zero here: https://docs.nvidia.com/cuda/npp/nppdefs.html#c.NppiBayerGridPosition. We can follow the same sequence. Can you update these files to have BGGR, then RGGB defined here.